op-node/rollup/types.go

```diff
@@ -585,81 +586,78 @@ func (c *Config) ActivationTimeFor(fork ForkName) *uint64 {
 	}
 }
 
+func (c *Config) SetActivationTime(fork ForkName, timestamp *uint64) {
```

Comment:
In the future, we only ever need to update this setter and getter and all other functions that have modular fork behavior automatically work for newer forks as well.